creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -119,6 +119,26 @@ spec:
       networkName: ovs-vlan-100
 ----
 
+It is also possible to define a multus network as the default pod network with
+https://github.com/intel/multus-cni[Multus]. A version of multus after this
+https://github.com/intel/multus-cni/pull/174[Pull Request] is required (currently master).
+
+[source,yaml]
+----
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: ovs-net
+          bridge: {}
+  networks:
+  - name: ovs-net
+    multus: # Multus network as default
+      default: true
```

Comment:
Can you please add a comment to explain that with this configuration the virt-launcher pod will not get the pod network.

Reply:
Done.